autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/sync/CredentialsSyncStore.kt

```diff
@@ -42,6 +63,20 @@ class RealCredentialsSyncStore @Inject constructor(private val context: Context)
     override var clientModifiedSince: String
         get() = preferences.getString(KEY_END_TIMESTAMP, "0") ?: "0"
         set(value) = preferences.edit(true) { putString(KEY_END_TIMESTAMP, value) }
+    override var isSyncPaused: Boolean
```

Comment:
Choose one, isSyncPaused or clientLimitExceeded. Like this it seems inconsistent.

Reply:
Want to leave it like this. For consumers, sync paused. Internally we hit a limit, we could have other errors that pause sync. Just to have flexibility.